capitolweb/workers/crec_parser.py

```diff
@@ -51,6 +60,49 @@ def xpath_parse(root, paths, namespaces):
     return results
 
 
+def attribute_segments(sents, speaker_ids, record_id):
+    previous = None
+    current = None
+    segment_index = 0
+    segment_sents = []
+    segments = []
+    individual_speakers = speaker_ids.keys()
```

Comment:
we get an empty list for `segments` if there are no speakers for the record; right?